docs/guide/templating/product_images.md

```diff
@@ -0,0 +1,51 @@
+# Product images and assets
+
+Product images and assets can be taken from the content model (if this data provider is used).
+External images can also be stored in the file system. 
+
+If images come from the file system, they have to be stored in the following folders (the folders can be configured):
```

Comment:
"the folders can be configured"
Does it mean that you can change the default name of the folder somewhere? If so, I'd create a note that says "You can change the default folder names by <procedure>.", and place such note under the list below.